src/BaGet.Core/Storage/FileStorageService.cs

```diff
@@ -33,7 +33,13 @@ public Task<Stream> GetAsync(string path, CancellationToken cancellationToken =
             cancellationToken.ThrowIfCancellationRequested();
 
             path = GetFullPath(path);
-            var content = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.Read);
+            Stream content = new MemoryStream();
+            try {
+                content = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.Read);
+            } catch (DirectoryNotFoundException e) {
+                Console.WriteLine("Error reading from the package folder (" + path + ") Please make sure your package folder starts with an uppercase letter. Refer to https://github.com/loic-sharma/BaGet/issues/298 for further reference.", ConsoleColor.Red);
```

Comment:
Why uppercase? Shouldn't this be lowercase?

Reply:
You're right, I got a little confused there. Seems alright 😆